### PR TITLE
Implemented Live Progress Bars for Data Export through debezium

### PR DESCRIPTION
**Pull request by sanyamsinghal**

### CODE REVIEW

**Comment by sanyamsinghal** [comment]
Oracle Automation Test results: https://jenkins.dev.yugabyte.com/job/users/job/yb-voyager-testing/job/yb-voyager-testing-pipeline/357/

File: yb-voyager/src/srcdb/mysql.go [hunk 1]
```diff
@@ -45,25 +45,18 @@ func (ms *MySQL) GetTableRowCount(tableName string) int64 {
...
-	if !tableProgressMetadata.IsPartition {
-		query = fmt.Sprintf("SELECT table_rows from information_schema.tables "+
-			"where table_name = '%s'", tableProgressMetadata.TableName.ObjectName.Unquoted)
-	} else {
-		query = fmt.Sprintf("SELECT table_rows from information_schema.partitions "+
-			"where table_name='%s' and partition_name='%s' and table_schema='%s'",
-			tableProgressMetadata.ParentTable, tableProgressMetadata.TableName.ObjectName.Unquoted, tableProgressMetadata.TableName.SchemaName.Unquoted)
-	}
+	query := fmt.Sprintf("SELECT table_rows from information_schema.tables "+
+		"where table_name = '%s'", tableName.ObjectName.Unquoted)
```
Comment (amit-yb) [inline yb-voyager/src/srcdb/mysql.go:51]: What if same table name is present in two different mysql databases?

File: yb-voyager/src/srcdb/oracle.go [hunk 2]
```diff
@@ -46,24 +46,18 @@ func (ora *Oracle) GetTableRowCount(tableName string) int64 {
...
-	var query string
-	if !tableProgressMetadata.IsPartition {
-		query = fmt.Sprintf("SELECT NUM_ROWS FROM ALL_TABLES "+
-			"WHERE TABLE_NAME='%s'", tableProgressMetadata.TableName.ObjectName.Unquoted)
-	} else {
-		query = fmt.Sprintf("SELECT NUM_ROWS FROM ALL_TAB_PARTITIONS "+
-			"WHERE TABLE_NAME='%s' AND PARTITION_NAME='%s'", tableProgressMetadata.ParentTable, tableProgressMetadata.TableName.ObjectName.Unquoted)
-	}
+	query := fmt.Sprintf("SELECT NUM_ROWS FROM ALL_TABLES "+
+		"WHERE TABLE_NAME='%s'", tableName.ObjectName.Unquoted)
```
Comment (amit-yb) [inline yb-voyager/src/srcdb/oracle.go:52]: Need to check the schema name?

**Review by amit-yb — APPROVED** [review]
LGTM with a question.

File: yb-voyager/src/srcdb/oracle.go (hunk 2 above)
Comment (sanyamsinghal) [reply yb-voyager/src/srcdb/oracle.go]: Yeah nice catch, fixed.

File: yb-voyager/src/srcdb/mysql.go (hunk 1 above)
Comment (sanyamsinghal) [reply yb-voyager/src/srcdb/mysql.go]: Yeah nice catch, fixed.

**Review by amit-yb — APPROVED** [review]
LGTM.

File: yb-voyager/src/dbzm/status.go [hunk 3]
```diff
@@ -68,3 +69,50 @@ func IsLiveMigrationInStreamingMode(exportDir string) bool {
...
+	return fmt.Sprintf("%s.%s", schemaOrDb, tableName)
+}
+
+func (status *ExportStatus) GetTableExportedRowCount(tableSno int) int64 {
+	for i := range status.Tables {
+		if status.Tables[i].Sno == tableSno {
+			return status.Tables[i].ExportedRowCountSnapshot
+		}
+	}
+	return 0
```
[the remark this answers is not in the document]
Comment (sanyamsinghal) [reply yb-voyager/src/dbzm/status.go]: done

**Pull request merged**